js/ooyala_ssai.js

```diff
@@ -328,9 +330,12 @@ OO.Ads.manager(function(_, $)
     this.pauseAd = function(ad)
     {
       //Removing the ad timeout since ad was paused
-      if (ad && ad.ad && ad.ad.data && this.adIdDictionary[ad.ad.data.id]) {
-        clearTimeout(this.adIdDictionary[ad.ad.data.id].adTimer);
-      }
+      if (adMode) {
+        _handleTrackingUrls(this.currentAd, ["pause"]);
+        if (ad && ad.ad && ad.ad.data && this.adIdDictionary[ad.ad.data.id]) {
+          clearTimeout(this.adIdDictionary[ad.ad.data.id].adTimer);
+        }
+  	  }
```

Comment:
Can we fix this indentation?

Reply:
@jpfranco there were mixed spaces and tabs, fixed now